algebird-core/src/main/scala/com/twitter/algebird/Hash128.scala

```diff
@@ -56,18 +66,20 @@ object Hash128 extends java.io.Serializable {
    * but has been more commonly used in HLL.
    */
   def murmur128Utf8String(seed: Long): Hash128[String] =
-    murmur128ArrayByte(seed).contramap(_.getBytes("UTF-8"))
+    murmur128ArrayByte().contramap(_.getBytes("UTF-8"))
 
   def murmur128Int(seed: Long): Hash128[Int] = new Hash128[Int] {
```

Comment:
I think we should rename `seed` to `defSeed` to avoid a collision in names below, and it should be the `defaultSeed` for the instance created.